DSLink.UWP/UWPPlatform.cs

```diff
@@ -0,0 +1,15 @@
+using DSLink.Connection;
+using PCLStorage;
+
+namespace DSLink.UWP
+{
+    public class UWPPlatform
+    {
+        public static void Initialize()
+        {
+            Configuration.StorageBaseFolder = FileSystem.Current.LocalStorage;
+            Websockets.Universal.WebsocketConnection.Link();
```

Comment:
This sets up platform specific code for Universal Windows, and all other platforms have it as well.